src/Config/ConfigOverrides.php

```diff
@@ -43,6 +56,21 @@ public function loadOverrides($names) {
         'front' => $this->state->get('stanford_profile.front_page'),
       ];
     }
+
+    // Add to the config ignore so that it will ignore the current theme's
+    // settings config.
+    if (in_array('config_ignore.settings', $names) && $this->configFactory) {
+      // We have to get the original values to add to the array.
+      $existing_ignored = $this->configFactory->getEditable('config_ignore.settings')
+        ->getOriginal('ignored_config_entities', FALSE);
+      $themes = $this->configFactory->getEditable('core.extension')
+        ->getOriginal('theme');
+      foreach (array_keys($themes) as $theme_name) {
+        $existing_ignored[] = "$theme_name.settings";
+      }
+      $overrides['config_ignore.settings']['ignored_config_entities'] = $existing_ignored;
+    }
```

Comment:
I suppose it would be possible for a site to have multiple sub-themes over time and to change from one to another. Wouldn't we want to get a list of all available themes and to ignore the settings for all of them? 

Scenario: 
* Site changes theme from stanford_basic to abc theme
* time goes on and changes theme to xyz theme which also has settings
* abc theme settings stay in DB but are not config ignored and we do a deployment
* abc theme settings get deleted

Reply:
Thats what this does. All enabled themes respective settings are ignored. Not just the currently active one. If a theme is Uninstaller thats different and will delete the settings for it anyways